dr_visibilities.py

```diff
@@ -693,7 +694,7 @@ def main(self):
                         active_op.start(self.station, chan0, navg, nchan, chan_bw, npol, pols)
                         was_active = True
                     active_op.write(time_tag, idata)
-                    
+                    self.client.write_monitor_point('latest_time_tag', time_tag)    
```

Comment:
This is probably fine for slow but I'm not sure about fast where there is a new gulp of data coming in every 100 ms and a write every 1 s.  That doesn't leave a lot of headroom for etcd communication problems and any slowness here ultimately kills the packet capture.

Reply:
I'll just do it for slow then.

Reply:
Is there any other information about the data stream that would be good to expose?

Reply:
Lots of that is in the written data so I think we're good. I just want to know which file was last written.